Sources/Swish/Models/SwishError.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+
+public enum SwishError {
+    case decodingError(Error)
+    case serverError(code: Int, data: Data?)
+    case urlSessionError(Error, response: HTTPURLResponse?)
+}
+
+extension SwishError: Error {}
+
+extension SwishError: CustomNSError {
+    public static let errorDomain = "com.thoughtbot.swish"
+
+    public var errorCode: Int {
+        switch self {
+        case .decodingError:
+            return 1
+        case .serverError:
+            return 4
+        case .urlSessionError:
+            return 5
+        }
+    }
+
+    #if !swift(>=3.1)
```

Comment:
Do we still support <3.1?

Reply:
Nope – I'll remove this as part of a separate PR